src/js/stores/__tests__/MesosStateStore-test.js

```diff
@@ -180,6 +180,52 @@ describe('MesosStateStore', function () {
     });
   });
 
+  describe('#getSchedulerTasks', function () {
+    beforeEach(function () {
+      this.get = MesosStateStore.get;
+      MesosStateStore.get = function () {
+        return {
+          frameworks: [{
+            name: 'marathon',
+            tasks: [
+              {
+                id: 1,
+                labels: [{key: 'DCOS_PACKAGE_FRAMEWORK_NAME', value: 'foo'}]
+              },
+              {
+                id: 2
+              },
+              {
+                id: 3,
+                labels: [{key: 'DCOS_PACKAGE_FRAMEWORK_NAME', value: 'bar'}]
+              }
```

Comment:
Could we also add some tasks that will not be included, to test out that scenario as well?